docs/_data/portland-2025-config.yaml

```
@@ -0,0 +1,286 @@
+
+shirts:
+  url: https://shirt.writethedocs.org/
+  ends: "May 31, 2025"
+
+date: # how do we handle these? human readable would be nice.
+  main: "**May 4-6, 2025**"
+  short: May 4-6, 2025
+  tickets_live: "December 2024"
+  month: May
+
+  # Conference is special for the index to show the combined conference days
+  conference:
+    event: Speaker talks
+    date: May 5-6
+    summary: The main days of the conference. Listen to a diverse panel of speakers share their insights and experience. See our <a href="https://www.writethedocs.org/conf/portland/2025/speakers/">full list of speakers</a>.
+    icon: conference
+
+  # These are listed in the schedule
+  day_one:
+    event: Hike
+    date: May 3
+    # summary: We encourage you to go on a hike in your own beautiful home town.
+    summary: If you're in town early, join us for the  <a href="/conf/portland/2025/hike/">Hike</a>. Portland has excellent access to the woods right in the city.
+    icon: hike
+    dotw: Saturday
+  day_two:
+    event: Writing Day & Reception
+    date: May 4
+    summary: Kick off the conference with <a href="/conf/portland/2025/writing-day/">Writing Day</a> and work with fellow documentarians on a project. Followed by the opening reception.
+    icon: writing
+    dotw: Sunday
+    writing_day_time: 9am-5pm
+    reception_time: 5-7pm
+  day_three:
+    event: Conference Day 1
+    date: May 5
+    summary: This is the main event! Hear from lots of interesting folks about all things documentation. We will have talks all day, and unconference sessions running in parallel.
+    dotw: Monday
+    talk_time: 9am-5pm
+    unconference_time: 9:45am-5pm
+    social_time: 6:30pm-8:30pm
+    icon: conference
+  day_four:
+    event: Conference Day 2
+    date: May 6
+    summary: And the conference goes on! More talks and unconference today, along with our Sponsor Expo!
+    dotw: Tuesday
+    talk_time: 9am-4pm
+    job_fair_time: 9-12pm
+    unconference_time: 1:35pm-4pm
+    icon: conversation
+  total_talk_days: 2
+
+about:
+  attendees: 400
+  summary:
+    "The main presentation track takes place from **May 5-6 (Monday and Tuesday) from 9am to 5pm**.
+    During the main event we also run an :doc:`/conf/portland/2025/unconference`."
+
+  social_venue:  TBD
+  venue: Revolution Hall
+  photos: TBD
+  mainroom: Main stage
+  unconfroom: Library & Astoria room (1st floor)
+  projector_ratio: "16:9"
+  job_fair_room: Sponsor area
+
+cfp:
+  url: https://pretalx.com/wtd-portland-2025/cfp  # Open November 6 (Mon) -> 9 weeks to submit
+  ends: "January 15, 2025"                        # (Mon) 1 week, to review. It'll take a little longer to announce
+  notification: "January 30, 2025"                # (Mon) 8 weeks to write and prepare
+  slides_by: "April 5, 2025"                      # (Mon) 1 week before
+  preview: "https://writethedocs-www--2098.org.readthedocs.build/conf/portland/2025/schedule/"                                 # Preview URL TODO
```

```suggestion
cfp:
  url: https://pretalx.com/wtd-portland-2025/cfp  # Open October 6 (Mon) -> 9 weeks to submit
  ends: "January 21, 2025"                        # (Mon) 1 week, to review. It'll take a little longer to announce
  notification: "February 04, 2025"                # (Mon) 8 weeks to write and prepare
  slides_by: "April 28, 2025"                      # (Mon) 1 week before
  preview: "TODO"                                 # Preview URL TODO
```

How do we feel about this? Too tight? Between notification and conference? Too long in the CFP?